### ISSUE TITLE
open_access.is_gold_journal

### ISSUE THREAD

**Issue by sckott** [ourresearch/journalsdb#28]
Sage

Is this journal correctly determined as Gold OA?  https://api.journalsdb.org/journals/1092-5872  - Looking at the journal website https://journals.sagepub.com/author-instructions/JIX it seems like OA is an option, so wouldn't it be a hybrid journal, and not Gold? It's not in DOAJ.

Springer

These Springer titles all seem to be hybrid/transformative journals, eventually becoming gold OA, but right now seems like they are hybrid? Or if it's a transformative journal, maybe you call it Gold OA? Some of these are in DOAJ and some are not. 

ISSN | Title | Is it Gold OA?
-- | -- | --
1976-8257 | Toxicological research | no, hybrid, transformative
0104-6632 | Brazilian Journal of Chemical Engineering | no, hybrid, transformative
2662-9283 | SN Social Sciences | no, hybrid, transformative
1976-4251 | Carbon letters | no, hybrid, transformative
2662-7655 | Systems microbiology and biomanufacturing | no, hybrid, transformative
0019-5413 | Indian journal of orthopaedics | no, hybrid, transformative
2454-9983 | Proceedings of the Indian National Science Academy | ??, page wouldn't resolve
1229-7801 | Journal Of The Korean Ceramic Society | no, hybrid, transformative
2358-1883 | Trends in Psychology | no, hybrid, transformative
0019-5235 | Indian journal of history of science | ??, page wouldn't resolve
2447-9462 | Journal of Sedimentary Environments | no, hybrid, transformative

**Comment by caseydm** [comment]
@richard-orr can you please help with this?

Right now Richard is maintaining the gold OA list and I pull it from his source. We discussed moving that into journalsdb at some point but have not done it yet.

**Comment by sckott** [comment]
Thanks both!

**Comment by richard-orr** [comment]
These should all be hybrid. A couple of years ago I labeled a bunch of non-DOAJ journals Gold based on 100% OA rates 
over a certain period and some are false positives from that. I removed these from the list.

Others are in DOAJ:

0104-6632, Brazilian Journal of Chemical Engineering, https://doaj.org/toc/0104-6632
2358-1883, Trends in Psychology, https://doaj.org/toc/2358-1883
2447-9462, Journal of Sedimentary Environments, https://doaj.org/toc/2447-9462

They all seem to have "reverse-flipped" by moving to Springer in 2020, so I took them off the DOAJ list.

I'm updating the oa_status of all the DOIs in these journals now. The journal-level stats will be recalculated and exported tomorrow morning.
